src/cli/Cli.spec.ts

```diff
@@ -704,6 +705,13 @@ describe('Cli', () => {
   });
 
   it(`shows validation error when no option from a required set is specified`, (done) => {
+    sinon.stub(cli, 'getSettingWithDefaultValue').callsFake(((settingName, defaultValue) => {
+      if (settingName === "prompt") { return false; }
```

Comment:
Hi @nicodecleyre, I think we should just return defaultValue in all scenarios. I mean: this method can only be stubbed once. Prompt=false is the default, so let's just return defaultValue. This also means we can write the stub as a oneliner, which is nice for readability.

This flies for all your added stubs I think? Maybe you can use multi-line search and replace to be done with it the fastest.

Reply:
cool, is nicer and more readable. The PR is updated!